test/tours.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable func-names */
```

Comment:
I prefer not to use these file-level disables. In that case, let's modify the .eslintrc, if we agree that we need to remove / add something.

Reply:
These only exists in test files. I think it is acceptable to keep them as they are. I can refactor to get rid of `func-names` exceptions in general. Let me know what you think.